schema-admin.graphql

```diff
@@ -119,13 +119,6 @@ type ScheduledSurface {
     prospectTypes: [ProspectType!]!
 }
 
-"""
-Temp type to pacify federation checks
-"""
-type ApprovedCuratedCorpusItem @key(fields: "url") {
-    url: Url!
-}
-
```

Comment:
Unrelated change - I think this is safe to remove now that Prospect API has been updated to use the shortened `ApprovedCorpusItem` entity instead.